tests/BlazorServerCallsGraph/Startup.cs

```diff
@@ -26,15 +26,13 @@ public Startup(IConfiguration configuration)
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            string[] scopes = Configuration.GetValue<string>("CalledApi:CalledApiScopes")?.Split(' ');
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
                     .AddMicrosoftIdentityWebApp(Configuration, "AzureAd")
-                        .CallsWebApi(scopes)
+                        .CallsWebApis()
+                            .AddMicrosoftGraphServiceClient(Configuration.GetSection("GraphBeta"))
+                            .AddDownstreamApiService("CalledApi", Configuration.GetSection("CalledApi"))
                         .AddInMemoryTokenCaches();
```

Comment:
That's the kind of code that you would have if you want to call several services.

If you want to use ITokenAquisition directly, just use `.CallsWebApis()`
Otherwise:
- if you want to use the MicrosoftGraphServiceClient wrapper, use `.AddMicrosoftGraphServiceClient` under `.CallsWebApis()`
- if you want to use one or several instances of the downstream API service (they have a name, which here, I've chosen the same as the section in the config file to make it simpler, but that's not absolutely necessary), you woud have one or several calls to `.AddDownstreamApiService("NameOfTheService", Configuration.GetSection("SectionInConfig"))`

Then in the controller, you'd use IDownstreamWebApi.CallsWebApiForUser("NameOfTheService", and have the possiblity to override the configuration from the config file with a delegate.